Address all reviewers comments from second revision

MichalisPanayides opened the pull request.

MichalisPanayides committed:
Add editor and reviewers comments

MichalisPanayides committed:
Add suggested references in lit review

MichalisPanayides committed:
Update known words

MichalisPanayides committed:
Apply suggestion by proselint

MichalisPanayides committed:
Add response to reviewers

drvinceknight commented on reviews/EJOR/02/ResponsetoReviewers.tex:
```diff
@@ -0,0 +1,102 @@
+
+    The authors would like to thank once again all the reviewers for their
+    constructive feedback and for
+    the time they took to review the manuscript.
+    Please find the summary of the changes made:
```
```suggestion
    the time they took to review the manuscript.
    We are also pleased to see that the reviewers feel that the revision has addressed their comments and that the manuscript presents scientifically correct and useful results.
    We have made the remaining requested minor changes and also provide a response to one of the 
    reviewers whose remaining concern is about the fit of this paper for EJOR.
```

drvinceknight commented on reviews/EJOR/02/ResponsetoReviewers.tex:
```diff
@@ -0,0 +1,102 @@
+        do is sufficient.
+    \end{itemize}
+
+    Here's a list of reasons of why I think this paper should be considered for
+    publication in EJOR.
+    Firstly, reviewer 2 states that the paper does not provide any insights.
+    We would like to point out that this paper does provide theoretical results
+    and insights.
+    Section 3 provides detailed theoretical expressions of the queueing model's
+    performance measures.
```
```suggestion
    We are grateful for the opportunity to respond to this as it seems that the 
    remaining concern is the fit of this work in EJOR and not the work itself.
    Firstly, this paper does provide theoretical results
    and insights.
    Section 3 provides detailed theoretical expressions of the queueing model's
    performance measures.
```

drvinceknight commented on reviews/EJOR/02/ResponsetoReviewers.tex:
```diff
@@ -0,0 +1,102 @@
+    A busy model is examined and insights on ways to increase the relative
+    efficiency of the model are presented.
+
+    Secondly here is a list of papers already published in EJOR that introduce
+    a new model and give details on numerical procedures for solving it but
+    without any new theorems:
+    \begin{itemize}
```
```suggestion
    We do understand the main point being raised by the reviewer in that this paper is not a purely theoretic one.
    For the benefits of the handling editor who we believe is left with the decision of whether or not the paper is in the scope of EJOR, here is a list of a number of papers published in EJOR that have a similar applied emphasis:
    \begin{itemize}
```

drvinceknight commented on reviews/EJOR/02/ResponsetoReviewers.tex:
```diff
@@ -0,0 +1,102 @@
+    a new model and give details on numerical procedures for solving it but
+    without any new theorems:
+    \begin{itemize}
+        \item A queueing model for general group screening policies and dynamic
+        item arrivals \\
+        https://doi.org/10.1016/j.ejor.2010.05.042
+        \item Selfish routing in public services: \\
+        https://doi.org/10.1016/j.ejor.2013.04.003
+        \item The effect of ambulance relocations on the performance of
+        ambulance service providers \\
+        https://doi.org/10.1016/j.ejor.2015.12.022
+        \item Robust and stochastic formulations for ambulance deployment and
+        dispatch \\
+        https://doi.org/10.1016/j.ejor.2019.05.011
+        \item A Markovian queueing model for ambulance offload delays \\
+        https://doi.org/10.1016/j.ejor.2012.11.030
+        \item Models and algorithms for an integrated vessel scheduling and tug
+        assignment problem within a canal harbor \\
+        https://doi.org/10.1016/j.ejor.2021.10.037
+        \item The application of operational research to European air traffic
+        flow management - understanding the context \\
+        https://doi.org/10.1016/S0377-2217(99)00084-3
```
Could you add the title/authors etc here. Just to make it less effort for the reviewer.

drvinceknight commented on reviews/EJOR/02/ResponsetoReviewers.tex:
```diff
@@ -0,0 +1,102 @@
+            \item Short Communications, if they correct important errors found in papers previously published in EJOR
+        \end{itemize}
+    }
+
```
I would add reviewer 3's one line comment and just say "Thank you" as a response.

MichalisPanayides committed:
Apply suggestions from code review

Co-authored-by: Vince Knight <vince@vknight.org>

MichalisPanayides committed:
Further minor changes

- Add main arguments for publishing our paper to editor
- Include authors, date published and Volume/issue on previous papers
- Add response to reviewer 3
- Minor rewords

MichalisPanayides committed:
Update known words

MichalisPanayides committed:
Update known words again

drvinceknight commented on reviews/EJOR/02/ResponsetoReviewers.tex:
```diff
@@ -0,0 +1,135 @@
+    Our main arguments are:
+    \begin{itemize}
+        \item Our paper provides theoretical results and insights
+        \item There is a number of pages in EJOR that are similar to ours
```
```suggestion
        \item There are a large number of papers in EJOR that are similar to ours
```

drvinceknight reviewed (changes requested):
Looks good @11michalis11 just the minor typo.

MichalisPanayides committed:
Fix minor typo

Co-authored-by: Vince Knight <vince@vknight.org>

MichalisPanayides committed:
Another minor typo
